win10/RemoteBlinky/RemoteBlinky/MainPage.xaml.cs

```diff
@@ -24,77 +12,77 @@ namespace RemoteBlinky
-                bluetooth = new BluetoothSerial( "RNBT-E072" );
+            arduino = App.Arduino;
+            App.Telemetry.TrackEvent( "RemoteBlinky_Windows10_SuccessfullyConnected" );
 
-                arduino = new RemoteDevice( bluetooth );
-                bluetooth.ConnectionEstablished += OnConnectionEstablished;
+            App.Arduino.DeviceConnectionLost += Arduino_OnDeviceConnectionLost;
 
-                //these parameters don't matter for bluetooth
-                bluetooth.begin( 0, 0 );
-            }
-            else
-            {
-                /*
-                 * I've written my Arduino device VID and PID as a parameter to the BluetoothSerial constructor. You should change this to your 
-                 * device VID and PID if using USB. You can also use the UsbSerial.listAvailableDevicesAsync() function to list
-                 * available devices, but that is not covered in this sample.
-                 */
-                usb = new UsbSerial( "VID_2341", "PID_0043" );   //I've written in my device D directly
-                
-                arduino = new RemoteDevice( usb );
-                usb.ConnectionEstablished += OnConnectionEstablished;
+            currentState = PinState.LOW;
+            OnButton.IsEnabled = true;
+            OffButton.IsEnabled = true;
+            BlinkButton.IsEnabled = true;
+        }
 
-                //SerialConfig.8N1 is the default config for Arduino devices over USB
-                usb.begin( 115200, SerialConfig.SERIAL_8N1 );
+        private void Arduino_OnDeviceConnectionLost( string message )
+        {
+            ConnectionStatusMessage.Text = "Your device connection was lost!";
+            
+            if( timer != null )
+            {
+                timer.Stop();
+                timer = null;
             }
 
+            OnButton.IsEnabled = false;
+            OffButton.IsEnabled = false;
+            BlinkButton.IsEnabled = false;
         }
 
-        private void OnConnectionEstablished()
+        private void OnButton_Click( object sender, RoutedEventArgs e )
         {
-            //enable the buttons on the UI thread!
-            var action = Dispatcher.RunAsync( Windows.UI.Core.CoreDispatcherPriority.Normal, new Windows.UI.Core.DispatchedHandler( () => {
-                OnButton.IsEnabled = true;
-                OffButton.IsEnabled = true;
-            }));
+            //turn the LED connected to pin 13 ON
+            currentState = PinState.HIGH;
+            arduino.digitalWrite( 13, currentState );
         }
 
-        private void OnButton_Click( object sender, RoutedEventArgs e )
+        private void OffButton_Click( object sender, RoutedEventArgs e )
         {
-            //turn the LED connected to pin 5 ON
-            arduino.digitalWrite( 5, PinState.HIGH );
+            //turn the LED connected to pin 13 OFF
+            currentState = PinState.LOW;
+            arduino.digitalWrite( 13, currentState );
```

Comment:
Use a const int value for the pin number to set and define it at the top of the class.